.github/codecov.yml

```diff
@@ -40,6 +40,45 @@ flag_management:
         target: auto # Let's decrease this later.
         threshold: 10
 
+  groups:
```

Comment:
I think we can safely remove this, it does not appear on codecov documentation.

Reply:
Dropped:

[66525c0](https://github.com/gnolang/gno/pull/3316/commits/66525c0487650114c49701a065e17db150ec5a88)